src/classes/ADDR_Seasonal_UTIL.cls

```diff
@@ -256,6 +256,7 @@ public with sharing class ADDR_Seasonal_UTIL {
 
         TDTM_Runnable.Dmlwrapper dmlWrapper = new TDTM_Runnable.Dmlwrapper();
         ADDR_Addresses_TDTM.refreshParentAndSiblings(mapAccIdAddr, dmlWrapper, null);
-        TDTM_TriggerHandler.processDML(dmlWrapper);
+        Savepoint sp = Database.setSavepoint();
```

Comment:
I'm attempting to simplify TDTM_TrigggerHandler. One of the things I'm doing is passing the savepoint as a parameter (I'm also removing the other savepoint, and instatiating TDTM_ObjectDataGateway inside the run method instead of passing it as a parameter every time - it was being passed as a parameter only because the original design had to alternatives for storing the TDTM config, one in custom setting and one in custom object).